tests/test_analytical.py

```diff
@@ -0,0 +1,58 @@
+import pytest
+import numpy as np
+
+from imkar import analytical
+
+"""
+References
+----------
+.. [#]J. Embrechts and A. Billon, "Theoretical Determination of the Random-
+Incidence Scattering Coefficients of Infinite Rigid Surfaces with a 
+Periodic Rectangular Roughness Profile", Acta Acustica united with Acustica,
+Bd. 97, Nr. 4, S. 607-617, Juli 2011, doi:10.3813/AAA.918441
+
+"""
+
+#Test compares the directional scattering coefficient under 30° with the one
+#shown in Figure 2 in [#]_.
+def test_rectangular_under_30_degree():
```

Comment:
test is also very slow, we could just test a few of this values, this might be sufficiant

Reply:
Reduced to 5 values.